Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM quay.io/wikiwatershed/taudem:5.3.8
+
+MAINTAINER Azavea <systems@azavea.com>
+
+COPY requirements.txt /tmp/
+COPY src/api/requirements.txt /tmp/src/api/
+RUN pip install --no-cache-dir -r /tmp/requirements.txt && \
+    pip install --no-cache-dir -r /tmp/src/api/requirements.txt && \
+    rm /tmp/requirements.txt && \
+    rm /tmp/src/api/requirements.txt
+
+ENV PYTHONPATH /usr/src:$PYTHONPATH
+
+COPY ./rwd_drb /usr/src/rwd_drb
+COPY ./rwd_nhd /usr/src/rwd_nhd
+COPY ./src/api /usr/src/api
+
+WORKDIR /usr/src/api
```

Comment:
For reference, here is the actual command that is used to spawn the RWD service in MMW: https://github.com/WikiWatershed/model-my-watershed/blob/565212ca1cbcfa37a242693359c1ff79fcb4e653/deployment/ansible/roles/model-my-watershed.rwd/templates/upstart-mmw-rwd.conf.j2#L19.